x-pack/plugins/reporting/export_types/common/execute_job/get_custom_logo.ts

```diff
@@ -4,14 +4,14 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { UI_SETTINGS_CUSTOM_PDF_LOGO } from '../../../common/constants';
-import { ConditionalHeaders, KbnServer, ReportingJob } from '../../../types';
+import { ConditionalHeaders, JobDocPayload, KbnServer } from '../../../types';
 
 export const getCustomLogo = async ({
   job,
   conditionalHeaders,
   server,
 }: {
-  job: ReportingJob;
+  job: JobDocPayload;
```

Comment:
Based on how much extra impact this caused, the rename from `ReportingJob` to `JobDocPayload` could go better if its in a standalone PR

Reply:
Agreed. I'll see how far I can carry this and see if this would be a deal breaker. I'd hate to try and go through history and segregate these changes into another PR. Going to treat this as a "nice-to-have"